README.md

```diff
@@ -127,6 +127,7 @@ Working examples
 
 Here are some repos that use cibuildwheel.
 
+- [Matplotlib](https://github.com/matplotlib/matplotlib) - The venerable Matplotlib, a Python library with C++ portions, built for Linux, Mac, and Windows on Github Actions.
 - [pyinstrument_cext](https://github.com/joerick/pyinstrument_cext) - A simple C extension, without external dependencies, built on Travis CI/Appveyor.
 - [websockets](https://github.com/aaugustin/websockets)
 - [Parselmouth](https://github.com/YannickJadoul/Parselmouth)
```

Comment:
```suggestion
- [Parselmouth](https://github.com/YannickJadoul/Parselmouth) - A pybind11-powered Python interface to the pre-existing Praat software, built with C++17 on GitHub Actions and sharing a build of the Python-independent core between different Python wheels in the same `cibuildwheel` run.
```

Reply:
Sure! Maybe also worth getting Cmake in there? :)

Here's my attempt-

> A Python interface to the Praat library, using pybind11, C++17 and Cmake, with the core Praat library shared between each wheel. All platforms built on Github Actions.

Reply:
OK, that's already closer, yes! Thanks! :-)

Let's just make sure it's clear I don't ship a shared library?

> A Python interface to the Praat library, using pybind11, C++17 and CMake, with the core Praat static library built only once and shared between wheels; all platforms built on Github Actions.

Reply:
sounds good

Reply:
Thanks! Let me push that and merge @nsoranzo's PR, then :-)

Reply:
@joerick, it still doesn't fit onto one line, when redering the markdown file:

https://github.com/joerick/cibuildwheel/blob/7f6812b13216dfd37912431036a0bd7e11af17f0/README.md

If you want to shrink the description go ahead; otherwise, I'm fine with this being merged, as well. I'll leave the decision up to you!

Reply:
Looks fine to me!